src/Controls/index.js

```diff
@@ -89,18 +96,18 @@ const setFullscreenControl = (map) => (
       position: 'topright'
     })
     .addTo(map)
-)
-
-const setLayerControls = (Config, DynamicLayerGroup, WMSLayerGroup, map) => {
+    )
+    
+const setLayerControls = (DynamicData, DynamicLayerGroup, WMSLayerGroup, map) => {
   const controlLayers = AddLayerControlsLayers()
-  const overlays = AddLayerControlsOverlays(
-    Config,
-    DynamicLayerGroup,
-    WMSLayerGroup,
-    map
-  )
-
-  Leaflet.control.layers(controlLayers, overlays).addTo(map)
+  const overlays = AddLayerControlsOverlays(DynamicData, DynamicLayerGroup, WMSLayerGroup, map)
+  const options = {
+    collapsed: true,
+    position: 'topright',
+    autoZIndex: true,
+    groupCheckboxes: true
```

Comment:
some of these are not required as the options have them defaulted, also can we not just defautl them to our choices so we dont need to pass in any options?